README.md

```diff
@@ -80,6 +85,38 @@ module depends on it.
 Known build and runtime issues are documented in the [SwC
 wiki](http://swc.lnls.br/).
 
+## CI Workflows
+
+Users of this repository for building IOC images can also take advantage of
+pre-defined continuous integration workflows in order to verify that images are
+built correctly after changes, and for uploading container images to the
+desired registry on Git tag creation.
+
+### GitHub Actions
+
+A YAML file must be added to the repository's `.github/workflows/` directory
+(e.g. `.github/workflows/build.yml`), with the following contents:
+
+```
+name: Build image
+on:
+  push:
+    tags:
+      - 'v*'
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build_and_push:
+    permissions:
+      packages: write
+      contents: read
+    uses: cnpem/epics-in-docker/.github/workflows/ioc-images.yml
+    secrets:
```

Comment:
I'm not specifying `ioc-images.yml@v0...` here. It seems like a bad idea to add yet another place where users need to update version identifiers on update.

Reply:
Sounds reasonable. That means will need to have the compromise to make it backward compatible, right?

Reply:
IMO breaking CI is not the end of the world. If we break anyone, they can then just pin the version if properly updating will be such a pain.

Reply:
Agreed; looks good.